### ISSUE TITLE
Menu on iPhone 6

### ISSUE THREAD

**Issue by jarnovanrhijn** [thoughtram/blog#84]
I noticed that the menu does not work on my iPhone 6. Not sure if on other mobile device it does work, Havent tried to debug it yet.

**Comment by PascalPrecht** [comment]
Can you please be a bit more specific what's not working?

It's hard to reproduce if we don't know what doesn't work.

**Comment by jarnovanrhijn** [comment]
Yeah sure, the problem is that i cant open de menu. Will check tommorow if i can debug it!!
